apps/emqx_bridge_http/src/emqx_bridge_http_connector.erl

```diff
@@ -329,7 +330,7 @@ on_query(
         {error, #{status_code := StatusCode}} ->
             ?SLOG(error, #{
                 msg => "http_connector_do_request_received_error_response",
-                note => "the body will be redacted due to security reasons",
+                note => "the request body will be redacted due to security reasons",
                 request => redact_request(NRequest),
```

Comment:
Seems that we already decided not to log request bodies, so I just chose to do that more consistently.

Also the redacted authn request is already logged in the authn provider itself.